src/ha-sankey-chart.ts

```diff
@@ -111,7 +112,14 @@ class SankeyChart extends SubscribeMixin(LitElement) {
             }
           }
           if (this.entityIds.length) {
-            const stats = await getStatistics(this.hass, data, this.entityIds);
+            const conversions: Conversions = {
```

Comment:
Please move the default values from here to `normalizeConfig` in `utils.ts`. I'd like to have all the  default values in one place

Reply:
Done